_includes/icon.liquid

```diff
@@ -0,0 +1,27 @@
+{% assign addIcon = include.icon %}
```

Comment:
Why not use `include.icon` directly in the code?

Reply:
I was thinking to store them in variables so I can facilitate using them in the frontmatter and user calls, I found a workaround to that and now I am using include.icon directly